pkg/controller/accept/accept_controller.go

```diff
@@ -0,0 +1,474 @@
+		kubeClient: kubeClient,
+		queue:      workqueue.NewNamedRateLimitingQueue(workqueue.DefaultControllerRateLimiter(), "master"),
+		logger:     logger,
+	}
+
+	machineSetInformer.Informer().AddEventHandler(cache.ResourceEventHandlerFuncs{
+		AddFunc:    c.addMachineSet,
+		UpdateFunc: c.updateMachineSet,
+	})
+
+	clusterInformer.Informer().AddEventHandler(cache.ResourceEventHandlerFuncs{
+		AddFunc:    c.addCluster,
+		UpdateFunc: c.updateCluster,
+	})
+
+	c.machineSetsLister = machineSetInformer.Lister()
+	c.machineSetsSynced = machineSetInformer.Informer().HasSynced
+
+	c.clustersLister = clusterInformer.Lister()
+	c.clustersSynced = clusterInformer.Informer().HasSynced
+
+	jobOwnerControl := &jobOwnerControl{controller: c}
+	c.jobControl = controller.NewJobControl(jobPrefix, machineSetKind, kubeClient, jobInformer.Lister(), jobOwnerControl, logger)
+	jobInformer.Informer().AddEventHandler(c.jobControl)
+	c.jobsSynced = jobInformer.Informer().HasSynced
+
+	c.jobSync = controller.NewJobSync(c.jobControl, &jobSyncStrategy{controller: c}, logger)
+
+	c.syncHandler = c.jobSync.Sync
+	c.enqueueCluster = c.enqueue
+	c.ansibleGenerator = ansible.NewJobGenerator(ansibleImage, ansibleImagePullPolicy)
+
+	return c
+}
+
+// Controller manages launching the master/control plane on machines
+// that are masters in the cluster.
+type Controller struct {
+	client     clusteroperatorclientset.Interface
+	kubeClient kubeclientset.Interface
+
+	// To allow injection of syncMachineSet for testing.
+	syncHandler func(hKey string) error
+
+	// To allow injection of mock ansible generator for testing
+	ansibleGenerator ansible.JobGenerator
+
+	jobControl controller.JobControl
+
+	jobSync controller.JobSync
+
+	// used for unit testing
+	enqueueCluster func(cluster *clusteroperator.Cluster)
+
+	// machineSetsLister is able to list/get machine sets and is populated by the shared informer passed to
+	// NewController.
+	machineSetsLister lister.MachineSetLister
+	// machineSetsSynced returns true if the machine set shared informer has been synced at least once.
+	// Added as a member to the struct to allow injection for testing.
+	machineSetsSynced cache.InformerSynced
+
+	// clustersLister is able to list/get clusters and is populated by the shared informer passed to
+	// NewController.
+	clustersLister lister.ClusterLister
+	// clustersSynced returns true if the cluster shared informer has been synced at least once.
+	// Added as a member to the struct to allow injection for testing.
+	clustersSynced cache.InformerSynced
+
+	// jobsSynced returns true of the job shared informer has been synced at least once.
+	jobsSynced cache.InformerSynced
+
+	// Machines that need to be synced
+	queue workqueue.RateLimitingInterface
+
+	logger log.FieldLogger
+}
+
+func (c *Controller) addMachineSet(obj interface{}) {
+	machineSet := obj.(*clusteroperator.MachineSet)
+	c.logger.Debugf("Enqueuing cluster for added machineset %s/%s", machineSet.Namespace, machineSet.Name)
```

Comment:
Checkout the logging pkg WithMachineSet and WithCluster functions, will keep all these standardized.